tests/test_basic_db_functions.py

```diff
@@ -0,0 +1,14 @@
+from fastapi.testclient import TestClient
+from server import app
+import re
+
+client = TestClient(app)
+
+def test_test_store_in_mongodb():
```

Comment:
this is not the right way to test for this.
if you want to test insert, you need to do 2 things:
1. load the data and see if the new value is inside
2. after the test is complete - delete the new data, to leave no side effects